src/main/java/org/trigon/hardware/rev/sparkecnoder/RelativeSparkEncoder.java

```diff
@@ -2,25 +2,36 @@
 
 import com.revrobotics.RelativeEncoder;
 
+/**
+ * A class that represents a relative encoder on a Spark MAX motor controller.
+ */
 public class RelativeSparkEncoder extends SparkEncoder {
     private final RelativeEncoder encoder;
 
+    /**
+     * Creates a new RelativeSparkEncoder.
+     *
+     * @param encoder the RelativeEncoder to use.
+     */
     public RelativeSparkEncoder(RelativeEncoder encoder) {
         this.encoder = encoder;
-        setConversionsFactor(1);
     }
 
+    /**
+     * Gets the position of the encoder in rotations.
+     *
+     * @return the position of the encoder in rotations.
+     */
     public double getPositionRotations() {
         return encoder.getPosition();
     }
 
+    /**
+     * Gets the velocity of the encoder in rotations per second.
+     *
+     * @return the velocity of the encoder in rotations per second.
+     */
     public double getVelocityRotationsPerSecond() {
         return encoder.getVelocity();
     }
-
-    @Override
-    public void setConversionsFactor(double conversionsFactor) {
-        encoder.setPositionConversionFactor(conversionsFactor);
-        encoder.setVelocityConversionFactor(conversionsFactor / 60);
-    }
```

Comment:
why was this removed?

Reply:
because they removed the setConversionFactorMethod

Reply:
talking about the different sim and real configs

Reply:
There isn't anything new for this?

Reply:
The javadocs still say it has it, but these are the only methods in the class. 
![image](https://github.com/user-attachments/assets/bee2e30f-0096-4cb8-ad01-f12093b84ad5)

Reply:
Ask on CD

Reply:
Probably moved to the config

Reply:
yeah it was